src/snmalloc/backend_helpers/largebuddyrange.h

```diff
@@ -134,6 +134,11 @@ namespace snmalloc
       return k;
     }
 
+    static void clear(Contents k)
```

Comment:
You mean adding an `IsZero' dimension on CapPtr? That is true but I'm tempted to leave it for future work...

Reply:
That's what I meant as the long-term goal; I agree it can be left as future work.  I just meant that if this were, instead...
```
static Contents clear(Contents k)
{
  return k;
}
```
and used as such, it'd be less disruptive when we do shift that to be `static Contents<Clear> clear(Contents<Unclear> k)` or whatever we go with.

Reply:
Ideally, it could take an r-value reference to a `Contents` as an argument, so that the caller has to explicitly move it and a static analyser will warn if you reference the old version.